WebHostLib/templates/playerOptions/macros.html

```diff
@@ -213,7 +213,7 @@
 {% endmacro %}
 
 {% macro RandomizeButton(option_name, option) %}
-    <div class="randomize-button" data-tooltip="Toggle randomization for this option!">
+    <div class="randomize-button" data-tooltip="Pick a random choice for this option.">
```

Comment:
For range option, I think it does not really make sense to pick a "choice". One could argue that all possible values are a choice, but I think it would make more sense (to my non-native English speaker mind) to call it directly a value, as that would also fit choice options.

```suggestion
    <div class="randomize-button" data-tooltip="Pick a random value for this option.">
```

Reply:
I personally think using "value" might make it more confusing to some vs choice? But it's sorta vibes based, so idk. If someone else chimes in too and says value better I'll probably change it

Reply:
"Pick a choice" sounds redundant, like saying "Choose a choice". "Pick a random value" or "Choose a random value" would be better.

Heck "Randomize this option." would probably shortcut the whole question.

Reply:
Wording like "Randomize this option" is exactly what we're trying to avoid here with the change lol

Reply:
I thought the operative word was "Toggle" and that was what was to be avoided.

There's also "Randomly choose for me!" if you want to make it more personable to the reader.

Reply:
Mainly, we want to avoid people thinking "if you don't click this then the option won't do anything"
ie: you have an option to make Precursor Orbs checks, and they want them to be checks, so the "Randomize this" button should surely make Precursor Orbs into checks